scripts/ES_requests.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2024, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import argparse
+import constant
+import socket
+import sys
+
+from functions.indexes.elasticsearch import get_eleasticsearch_host
+from functions.indexes.elasticsearch import check_status_eleasticsearch
+from functions.indexes.elasticsearch import request_elasticsearch
+from functions.plateform.plateform import get_plateform_info
+
+#
+# @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+#
+
+# Elastic REST API
+# cf. https://www.elastic.co/guide/en/elasticsearch/reference/current/rest-apis.html
+
+parser = argparse.ArgumentParser(description='Send requests (REST APIs) to the ES server', )
+     
+parser.add_argument("-m", "--method", help="request method to execute on ES server", required=True)
+parser.add_argument("-u", "--url", help="request url to execute on ES server", required=True)
+                                                  
+args = parser.parse_args()
+request_method = args.method
+request_url = args.url
+
+print("---------------------------------------------------------")
+print("ES request script")
+
+print("\n")
+# Just getting an enlightening url opportunistically from here because it exists
+# TODO better ?
+plateformName = get_plateform_info()['redirect_uri']
+elasticsearch_host = get_eleasticsearch_host(constant.DIRECTORY_SERVER_HOSTNAME)
+# TODO don't parse here, instead have the server return structured information
+elasticsearch_ip = socket.gethostbyname(elasticsearch_host.split(':')[0])
+# TODO we force http but should get this protocol from the server, some servers are not exposed on http but only https for example
+elasticsearch_url = constant.HTTP_PROTOCOL + elasticsearch_host
```

Comment:
Can't this all be in a separate function as we duplicate from elsewhere ? would simplify this script